src/main/index.ts

```diff
@@ -112,6 +114,12 @@ async function bootstrap() {
     }
 }
 
+// MacOS only event
+export var darwinURL: string | undefined;
```

Comment:
you will be sacrificed to the flying spaghetti monster soon if you don't change this var to a let
```suggestion
export let darwinURL: string | undefined;
```